File: 백준/Silver/11399. ATM/ATM.py
#사람의 수
N = int(input())

#인출에 걸리는 시간
P = list(map(int, input().split()))

#정렬
P.sort()
result = 0

for i in range(1, len(P)+1):
    result += sum(P[:i])

print(result)

File: 백준/Silver/11399. ATM/ATM.js
const input = require('fs').readFileSync(process.platform === 'linux' ? '/dev/stdin' : 'input.txt').toString().trim().split('\n')

const people = input[1].split(' ').map(Number).sort((a,b) => a-b);
function totalTime(t) {
    if(t == 0) return people[0];
    people[t] += totalTime(t-1);
    return people[t];
}
totalTime(Number(input[0])-1)
let answer = people.reduce((a,b) => a+b)
console.log(answer)